implementations/rust/ockam/ockam_api/src/cloud/project.rs

```diff
@@ -220,12 +221,14 @@ impl<'a> CreateProject<'a> {
         enforce_credentials: Option<bool>,
         users: &'a [T],
         services: &'a [T],
+        identity_name: Option<String>,
```

Comment:
You will have to change this to something like `Option<&'a T>`.

Reply:
Shouldn't it be `Option<CowStr<'a>>` like in the previous comment as well? The compiler keeps giving an error that these two should match (i.e. lines 215 and 224).

Reply:
Oh, sorry, it should be `Option<S>`, I got confused with the traits of the method definition.

Then you can transform that into a `Option<Cow<str>>` with `identity_name.map(|x| x.into())`.

Reply:
Thanks! That worked out fine.